<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileGeneratorDialogForm</class>
 <widget class="QDialog" name="TileGeneratorDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tile Generator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_15">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_14">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">QWidget { background: white }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="m_titleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Creates a set of tiles using the selected layers from application...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Input Extent</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_16">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <widget class="QToolButton" name="m_toolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>LL X:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="m_llxLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>LL Y:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="m_llyLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>UR X:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="m_urxLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>UR Y:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="m_uryLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Configuration Parameters</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Zoom Min:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="m_zoomMinSpinBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Zoom Max:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="m_zoomMaxSpinBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Tile Size:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLineEdit" name="m_tileSizeLineEdit">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="m_rasterRadioButton">
            <property name="text">
             <string>Tiles from Raster (uses the last level to create tiles)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="m_vectorRadioButton">
            <property name="text">
             <string>Tiles from Vector (uses always the original data to create tiles)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Output Parameters</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QToolButton" name="m_dirToolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="m_dirLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Format:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="m_formatComboBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="5">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="m_helpPushButton">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="m_okPushButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="m_cancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="m_validatePushButton">
         <property name="text">
          <string>Validate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_llxLineEdit</tabstop>
  <tabstop>m_llyLineEdit</tabstop>
  <tabstop>m_urxLineEdit</tabstop>
  <tabstop>m_uryLineEdit</tabstop>
  <tabstop>m_zoomMinSpinBox</tabstop>
  <tabstop>m_zoomMaxSpinBox</tabstop>
  <tabstop>m_tileSizeLineEdit</tabstop>
  <tabstop>m_rasterRadioButton</tabstop>
  <tabstop>m_vectorRadioButton</tabstop>
  <tabstop>m_dirToolButton</tabstop>
  <tabstop>m_dirLineEdit</tabstop>
  <tabstop>m_formatComboBox</tabstop>
  <tabstop>m_okPushButton</tabstop>
  <tabstop>m_validatePushButton</tabstop>
  <tabstop>m_cancelPushButton</tabstop>
  <tabstop>m_helpPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_cancelPushButton</sender>
   <signal>released()</signal>
   <receiver>TileGeneratorDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
